hnn_core/network.py

```diff
@@ -509,8 +554,134 @@ def plot_input(self, ax=None, show=True):
             plt.show()
         return ax.get_figure()
 
-    def plot_spikes(self, ax=None, show=True):
-        """Plot the spiking activity for each cell type.
+
+class Spikes(object):
+    '''The Spikes class.
+
+    Parameters
+    ----------
+    times : list [n_trials, ] of list of float | None
+        Each element of the 1st-order list is a trial.
+        The 2nd-order list contains the time stamps of spikes.
+    gids : list [n_trials, ] of list of float | None
+        Each element of the 1st-order list is a trial.
+        The 2nd-order list contains the cell IDs of neurons that
+        spiked.
+    types : list [n_trials, ] of list of float | None
+        Each element of the 1st-order list is a trial.
+        The 2nd-order list contains the type of spike (e.g., evprox1
+        or L2_pyramidal) that occured at the corresonding time stamp.
+        Each gid corresponds to a type via Network::gid_dict.
```

Comment:
```suggestion
        Each gid corresponds to a type via Network().gid_dict.
```

isn't that convention from C or C++?

Reply:
Oh yeah, my bad.